docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -1030,7 +1030,7 @@ Status Codes:
 
 `POST /containers/(id)/update`
 
-Update resource configs of one or more containers.
+Update configs of one or more containers.
```

Comment:
I think "update configuration of one or more containers"  ("configuration" instead of "configs")